content/tutorial-nlp-from-scratch.md

```diff
@@ -0,0 +1,848 @@
+
+Along with data, there are also growing concerns that data processing algorithms are influencing policy and daily lives in ways that are not transparent and introduce biases. Certain biases such as the [Inductive Bias](https://en.wikipedia.org/wiki/Inductive_bias#:~:text=The%20inductive%20bias%20(also%20known,that%20it%20has%20not%20encountered.&text=The%20kind%20of%20necessary%20assumptions,in%20the%20phrase%20inductive%20bias) are absolutely essential to help a Machine Learning model generalise better, for example the LSTM we built earlier is biased towards preserving contextual information over long sequences which makes it so suitable for processing sequential data. The problem arises when [societal biases](https://hbr.org/2019/10/what-do-we-do-about-the-biases-in-ai) creep into algorithmic predictions. Optimizing Machine algorithms via methods like [hyperparameter tuning](https://en.wikipedia.org/wiki/Hyperparameter_optimization) can then further amplify these biases by learning every bit of information in the data. 
+
+
+There are also cases where bias is only in the output and not the inputs (data, algorithm). For example, in sentiment analysis [accuracy tends to be higher on female-authored texts than on male-authored ones]( https://doi.org/10.3390/electronics9020374). End users of sentiment analysis should be aware that its small gender biases can affect the conclusions drawn from it and apply correction factors when necessary. Hence, it is important that demands for algorithmic accountability should include the ability to test the outputs of a system, including the ability to drill down into different user groups by gender, ethnicity and other characteristics, to identify, and hopefully suggest corrections for, system output biases.
+
++++
+
+### Next Steps
+
++++
+
+You have learned how to build and train a simple Long Short Term Memory network from scratch using just NumPy to perform sentiment analysis.
+
+To further enhance and optimize your neural network model, you can consider one of a mixture of the following:
+
+- Increase the training sample size by increasing the `split_percentile`.
+- Alter the architecture by introducing multiple LSTM layers to make the network deeper.
+- Use a higher epoch size to train longer and add more regularization techniques, such as early stopping, to prevent overfitting.
+- Introduce a validation set for an unbiased valuation of the model fit.
+- Apply batch normalization for faster and more stable training.
+- Tune other parameters, such as the learning rate and hidden layer size.
+- Initialise weights using [Xavier Initialisation](https://d2l.ai/chapter_multilayer-perceptrons/numerical-stability-and-init.html) to prevent vanishing/exploding gradients instead of initialising them randomly.
```

Comment:
Maybe you wanted to link directly to https://d2l.ai/chapter_multilayer-perceptrons/numerical-stability-and-init.html#xavier-initialization?